### DISCUSSION TITLE
PlayCover on Intel?

### DISCUSSION THREAD

**Discussion by Yaewoo** [PlayCover/PlayCover#670] [Q&A] [answered]
Hello, just a question, can I run playcover on 2.6 GHz 6-Core Intel Core i7? Or is there any alternative to run genshin on it?

**Comment by Depal1** [comment, marked as answer]
No, PlayCover will only run on arm Macs. You could try running it through an Android emulator or perhaps through bootcamp (Windows on Mac). Running the game through bootcamp might be too taxing for Intel integrated graphics though, so it is only viable if you have a graphics card.
